testing/import_cmd/integration_test.go

```diff
@@ -74,60 +83,93 @@ func onlyRunForEmulatorTest(t *testing.T) {
+			dbName:       "menagerie",
+			wantErr:      false,
+			sourceFormat: constants.MYSQLDUMP,
 		},
 		{
-			name:    "employees dump file",
-			dumpUri: "gs://smt-integration-test/import/mysql/employees.sql",
-			dbName:  "employees_dump_file_mysql_test",
-			wantErr: false,
+			name:         "employees dump file",
+			dumpUri:      "gs://smt-integration-test/import/mysql/employees.sql",
+			dbName:       "employees_dump_file_mysql_test",
+			wantErr:      false,
+			sourceFormat: constants.MYSQLDUMP,
+		},
+		{
+			name:         "pagila",
+			dumpUri:      "../../test_data/pagila.sql",
+			dbName:       "pagila",
+			wantErr:      false,
+			sourceFormat: constants.PGDUMP,
+		},
+		{
+			name:         "pg_world",
+			dumpUri:      "../../test_data/pg_world.sql",
+			dbName:       "pg_world",
+			wantErr:      false,
+			sourceFormat: constants.PGDUMP,
+		},
+		{
+			name:         "adventureworks",
+			dumpUri:      "gs://smt-integration-test/import/pg_dump/Adventureworks.sql",
+			dbName:       "adventureworks",
+			wantErr:      false,
+			sourceFormat: constants.PGDUMP,
+		},
+		{
+			name:         "pg_first_time_data_load_small",
+			dumpUri:      "gs://smt-integration-test/import/pg_dump/pg_first_time_data_load_small.sql",
+			dbName:       "pg_ftdl_load_small",
+			wantErr:      false,
+			sourceFormat: constants.PGDUMP,
 		},
 	}
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
+			executeImportDump(t, constants.DIALECT_GOOGLESQL, tt)
+			executeImportDump(t, constants.DIALECT_POSTGRESQL, tt)
+		})
+	}
+}
 
-			dbURI := fmt.Sprintf("projects/%s/instances/%s/databases/%s", projectID, instanceID, tt.dbName)
-			log.Printf("Spanner database used for testing: %s", dbURI)
+func executeImportDump(t *testing.T, dialect string, testData testStruct) {
+	dbURI := fmt.Sprintf("projects/%s/instances/%s/databases/%s", projectID, instanceID, testData.dbName)
+	log.Printf("Spanner database used for testing: %s", dbURI)
 
-			createSpannerDatabase(t, projectID, instanceID, tt.dbName)
-			defer databaseAdmin.DropDatabase(ctx, &databasepb.DropDatabaseRequest{Database: dbURI})
+	defer databaseAdmin.DropDatabase(ctx, &databasepb.DropDatabaseRequest{Database: dbURI})
 
-			dumpFilePath := tt.dumpUri
+	dumpFilePath := testData.dumpUri
 
-			args := fmt.Sprintf("import -source-format=mysqldump -project=%s -instance=%s -database=%s -source-uri=%s",
-				projectID, instanceID, tt.dbName, dumpFilePath)
-			err := common.RunCommand(args, projectID)
-			assert.NoError(t, err)
+	args := fmt.Sprintf(
+		"import -source-format=%s -project=%s -instance=%s -database=%s "+
+			"-source-uri=%s -database-dialect=%s",
+		testData.sourceFormat, projectID, instanceID, testData.dbName, dumpFilePath, dialect)
+	fmt.Printf("Executing: %s\n", args)
+	err := common.RunCommand(args, projectID)
+	assert.NoError(t, err)
 
-			// TODO validation to be added.
-		})
-	}
+	// TODO validation to be added.
```

Comment:
why are we just importing the db without any validations?

Reply:
For validations, we have unit tests. This is just to ensure that there are no errors while importing data.